src/redux/books/books.js

```diff
@@ -0,0 +1,51 @@
+import { uuid4 } from 'uuid4';
```

Comment:
**You did a great job! However, there is one part that needs to be changed to make your code approve.**

- [ ] That part of your code is not created as it is meant to be. You need to get values (onChange) by updating localStates with (useState). Kindly change this implementation from DOM to more React way 😉
More info: [highlight](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=Set%20the%20data%20inputs%20in%20the%20local%20React%20state%20(set%20title%20and%20author%20%2D%20NOTE%3A%20categories%20and%20comments%20are%20NOT%20part%20of%20this%20step).)

<img width="612" alt="Screenshot 2022-12-16 at 02 24 40" src="https://user-images.githubusercontent.com/69011963/207970214-d7883dcf-a00a-4986-8612-c8a10802fcb2.png">